src/AWS.Deploy.Recipes/CdkTemplates/AspNetAppElasticBeanstalkLinux/Generated/Recipe.cs

```diff
@@ -214,6 +216,18 @@ private void ConfigureBeanstalkEnvironment(Configuration settings)
                 });
             }
 
+            if (!string.IsNullOrEmpty(settings.ReverseProxy))
+            {
+                optionSettingProperties.Add(
+                    new CfnEnvironment.OptionSettingProperty
+                    {
+                        Namespace = "aws:elasticbeanstalk:environment:proxy",
+                        OptionName = "ProxyServer",
+                        Value = settings.ReverseProxy
```

Comment:
If a user selects _"none"_ as the reverse proxy, would we still set an explicit "none" string as the `Value` ?

Reply:
Yes, the possible values are "nginx" and "none"

Reply:
Is the "none" string supported by  CDK and CloudFormation?